test/forms/declaration/CommodityDetailsSpec.scala

```diff
@@ -25,81 +25,86 @@ import play.api.libs.json.{JsObject, JsString}
 class CommodityDetailsSpec extends WordSpec with MustMatchers {
   import CommodityDetailsSpec._
 
-  "CommodityDetails mapping used for standard declaration" should {
+  "CommodityDetails mapping used for declarations where code required" should {
 
-    "return form with errors" when {
-      "provided with invalid commodity code" in {
-        val form = CommodityDetails.form(DeclarationType.STANDARD).bind(formData("A1234", "description"))
+    for (decType <- Set(DeclarationType.STANDARD, DeclarationType.SUPPLEMENTARY)) {
```

Comment:
Hmm, why in `CommodityDetails` we're handling `SIMPLIFIED` and `OCCASIONAL`, but spec is for `STANDARD` and `SUPPLEMENTARY`?

Reply:
The test is doing two sets of dec types